platform-server-rest/src/main/java/uno/cod/platform/server/rest/controller/CodingcontestController.java

```diff
@@ -47,4 +52,11 @@ public void activate(@PathVariable("uuid") UUID uuid) {
     public ContestInfoDto getResults(@PathVariable("uuid") UUID uuid) {
         return service.getResults(uuid);
     }
+
+    @AllowedForAdmin
+    @RequestMapping(value = "/contestuploadzip", method = RequestMethod.POST)
+    public ResponseEntity<String> contestuploadzip(@RequestParam("file") MultipartFile file) throws IOException {
+        service.createChallengeTemplateFromGameResources(file, sessionService.getActiveOrganization());
```

Comment:
FYI: `SessionService.getActiveOrgnization()` will be removed soon, as @Exordian is already working on this.